diff --git a/node_modules/unload/dist/es/browser.js b/node_modules/unload/dist/es/browser.js
index 2dbdc02..fa534d1 100644
--- a/node_modules/unload/dist/es/browser.js
+++ b/node_modules/unload/dist/es/browser.js
@@ -1,5 +1,6 @@
 /* global WorkerGlobalScope */
 function add(fn) {
+  return;
   if (typeof WorkerGlobalScope === 'function' && self instanceof WorkerGlobalScope) {// this is run inside of a webworker
   } else {
     /**
